packages/Path-View-Core.package/SClassDiagram.class/instance/dragBottomBy..st

```diff
@@ -0,0 +1,5 @@
+geometry
+dragBottomBy: aValue
```

Comment:
yep -- though its used after the window was resized it could be named #resizeBottomBy?